src/app/(views)/managed-courses/page.tsx

```diff
@@ -19,12 +19,23 @@ import toast from '@/utils/toast'
 import { Loading } from '@/components'
 import { cilPlus } from '@coreui/icons'
 import CIcon from '@coreui/icons-react'
+import ReactPaginate from 'react-paginate'
 
 export default function ManagedCourses() {
   const [deletingCourse, setDeletingCourse] = useState(false)
   const { courses, setCourses, fetchingCourses } = useGetCourses({ type: 'managed' })
   const [isDeleteModalVisible, setIsDeleteModalVisible] = useState(false)
 
+  const itemsPerPage = 8
+  const [currentPage, setCurrentPage] = useState(0)
+
+  const handlePageClick = (selectedPage: { selected: React.SetStateAction<number> }) => {
```

Comment:
Either:
1. Put this inside a useCallback. Read: https://www.w3schools.com/react/react_usecallback.asp; or
2. Just use setCurrentPage, no need to nest it inside another function - you wrote a function whose only purpose is to invoke another function. The only benefit we can get from this is that `handlePageClick` has a more descriptive name, which a good benefit.